source/docs/zero-to-robot/step-3/radio-programming.rst

```diff
@@ -2,198 +2,133 @@
 
-### Programmed Configuration
+4. Click :guilabel:`Browse...` and select the firmware file you downloaded.
 
-.. image:: /docs/hardware/hardware-basics/images/status-lights/openmesh-radio-status-lights.png
-  :alt: Lists the names of each of the status lights on the two legal radios.
+5. Click the :guilabel:`Upload` button.
 
-The Radio Configuration Utility programs a number of configuration settings into the radio when run. These settings apply to the radio in all modes (including at events). These include:
+.. warning:: The radio will take approximately 2-3 minutes to complete firmware updates. Do not remove power during this process. Damage to the radio can occur.  When the PWR light is solid and the SYS light is slowly blinking at 1 Hz, the firmware upgrade process is complete.
 
-- Set a static IP of ``10.TE.AM.1``
-- Set an alternate IP on the wired side of ``192.168.1.1`` for future programming
-- Bridge the wired ports so they may be used interchangeably
-- The LED configuration noted in the graphic above.
-- 4Mb/s bandwidth limit on the outbound side of the wireless interface (may be disabled for home use)
-- QoS rules for internal packet prioritization (affects internal buffer and which packets to discard if bandwidth limit is reached). These rules are:
+## Robot Radio Configuration (All Teams)
 
-  - Robot Control and Status (UDP ``1110``, ``1115``, ``1150``)
-  - Robot TCP & :term:`NetworkTables` (TCP ``1735``, ``1740``)
-  - Bulk (All other traffic). (disabled if BW limit is disabled)
+.. image:: images/radio-programming/configuration-page.png
+  :alt: The top section of the radio configuration page
 
-- :term:`DHCP` server enabled. Serves out:
+This section is used for configuring the VH-109 radio outside of competition. At competition, configuration will be done by a provided computer and manual configuration using this page **should not be used**.
 
-  - ``10.TE.AM.11`` - ``10.TE.AM.111`` on the wired side
-  - ``10.TE.AM.138`` - ``10.TE.AM.237`` on the wireless side
-  - Subnet mask of ``255.255.255.0``
-  - Broadcast address ``10.TE.AM.255``
+1. Select :guilabel:`Robot Radio Mode`
 
-- DNS server enabled. DNS server IP and domain suffix (``.lan``) are served as part of the DHCP.
+2. Enter the team number
 
-At home only:
+3. Enter the suffix, if desired.  This will help identify your robot and distinguish it from other networks.
 
-- SSID may have a "Robot Name" appended to the team number to distinguish multiple networks.
-- Firewall option may be enabled to mimic the field firewall rules (open ports may be found in the Game Manual)
+4. Enter the 6 GHz WPA/SAE key.  This key will need to match the key on the Access Point you configure.
 
-.. warning:: It is not possible to modify the configuration manually.
+5. Enter the 2.4 GHz WPA/SAE key.  This is the password team members will type in when connecting to the 2.4 GHz network, if available.
```

Comment:
What is this used for?

Reply:
It is a required input.  If the 3rd dip switch is turned on to enable the 2.4 AP this will be the password the user needs to enter to gain access to the device.